README.md

```diff
@@ -1,6 +1,7 @@
 # Kubernetes (K8s)
 
-[![GoPkg Widget]][GoPkg] [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/569/badge)](https://bestpractices.coreinfrastructure.org/projects/569)
+[![GoPkg Widget]][GoPkg] [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/569/badge)](https://bestpractices.coreinfrastructure.org/projects/569) [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fkubernetes%2Fkubernetes.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fkubernetes%2Fkubernetes?ref=badge_shield)
```

Comment:
what are the implications of embedding an image from another domain? does that expose people visiting this repo to tracking/metrics from that site?

Reply:
good point @liggitt 

Also looks like we are not ready to do this yet? see slack conversation with Ihor etc at:
https://kubernetes.slack.com/archives/C01672LSZL0/p1644869015702139
https://github.com/cncf/clomonitor/issues/50

@SubhasmitaSw please more the FOSSA Status for now.

Reply:
> I'm not very familiar with clomonitor.io ... do we think it's reasonable to require all OSS projects to add this many root dir files and a badge to the readme to get a good score?

The typical trend that I observed from other 100% scored projects on cloMonitor dashboard had these implementations in common.

Reply:
answering my own question... for the privacy bit at least, it looks like github scrapes it and serves it through a `https://camo.githubusercontent.com/...` domain